dcs-dtc/Models/AH64/Waypoints/Waypoint.cs

```diff
@@ -28,8 +28,18 @@ public Waypoint(int seq, string type, string ident, string free, string mgrs,int
         public static Waypoint FromStrings(string type, string ident, string free, string mgrs, string elevation)
         {
             var match = MGRSRegex.Match(mgrs);
-            var wpt = new Waypoint(0, type, ident, free, mgrs, int.Parse(elevation));
-            return wpt;
+            try

```

Comment:
This block could be simplified to
```
int? elev = null;
int.TryParse(elevation, elev)
return new Waypoint(0, type, ident, free, mgrs, elev);
```

Also please remove the commented out lines at the end